### ISSUE TITLE
frontmatter: resemble documented shortcode style

### ISSUE THREAD

**Issue by postb99** [McShelby/hugo-theme-relearn#672]
Hello,

Sorry for not being sure whether it's a bug or a regression over time, but in the following file https://github.com/McShelby/hugo-theme-relearn/blob/main/vscode-frontmatter/snippets.en.json and also snippets.de.json I noticed a typo: I see many "body" contents starting and ending with `{{<` and `>}}` rather than `{{%` and `%}}`.

Can you fix this?

Another (useful?) thing, the include snippet would uses shorter syntax, similar to your exampleSite :
`{{% include "basics/CHANGELOG.md" true %}}` rather than `{{% include file="basics/CHANGELOG.md" hidefirstheading="true" %}}` 

Using Hugo Extended 0.119.

**Comment by McShelby** [comment]
Thanks for pointing this out.

While the usage of `<` vs. `%` often depends on the use case, it should at least resemble the style used in the documentation.

I decided for not using positional parameters even for the simplest shortcode, because the snippets are used to auto generate the shortcode calls in your pages. If you later edit a page without the help of the VSCode front-matter extension, it should clearly communicate what each parameter does.

I also added the missing `highlight` shortcode with curated parameter from Hugo`s default implementation.
